### ISSUE TITLE
[Request] Better filename

### ISSUE THREAD

**Issue by JasperSurmont** [GrandTheftWalrus/RuneLite-World-Heatmap#9]
Wouldn't it make more sense to, instead of the userId, write the files with the date (+time) as filename? Perhaps in a directory with the userId? 

Also, the files written start with a minus (-) for me, which command line applications cannot handle too well (so I had to rename it first).

**Comment by GrandTheftWalrus** [comment]
This functionality might have been broken at some point but the plugin is supposed to keep backups which are named after user ID and datetime in a `backups` folder. User ID is used in the filenames because people can log into different accounts on the same machine and I wanted to keep em separated (also I didn't use OSRS usernames because they can change over time for an account). For some reason, OSRS user IDs all seem to be negative numbers, and I kept them as-is because minuses are legal filename characters and can just be surrounded with quotations if needed

**Comment by JasperSurmont** [comment]
Okay, that makes sense.
For me all pictures are just stored in the folders without any date / time in the name, which I assume is also causing it to overwrite previous images. This makes the autosave feature not really work since you need to edit the filename each time. 

I can take a look if you want why the date is being omitted?

I think the best structure would be:

```
worldheatmap
 | Heatmap Files
   | UserID
     | YYMMDD
       - Files
 | Heatmap Images
   | UserID
     | YYMMDD
       - Files
```

Any thoughts?

**Comment by GrandTheftWalrus** [comment]
> Okay, that makes sense. For me all pictures are just stored in the folders without any date / time in the name, which I assume is also causing it to overwrite previous images. This makes the autosave feature not really work since you need to edit the filename each time.
> 
> I can take a look if you want why the date is being omitted?

The autosave feature is only for the .heatmaps files, not for images, and they should be being stashed in this folder:
![image](https://github.com/GrandTheftWalrus/RuneLite-World-Heatmap/assets/70998757/9d73fa60-fe0a-4873-ba82-64a417fe2aac)
I don't have any recent files inside my backup folder though, which leads me to believe that I might have broken the autosave feature at some point in time. I'll shrek that out when I get some time (or you can see why it's busted if you're pinterested). There should be files in it that are named something like `[userId]_yyyy-mm-dd-hh-ss.heatmaps`.

> I think the best structure would be:
> 
> ```
> worldheatmap
>  | Heatmap Files
>    | UserID
>      | YYMMDD
>        - Files
>  | Heatmap Images
>    | UserID
>      | YYMMDD
>        - Files
> ```
> 
> Any thoughts?

Yee I suppose that would be better for organization/finding files, each account having its own folders for heatmap files/images

**Comment by JasperSurmont** [comment]
I'll look at it tonight and see how far I get.

**Comment by JasperSurmont** [comment]
Closed with #10
